### DISCUSSION TITLE
so-import-pcap, incomplete pcap imported, [packet size limited during capture]

### DISCUSSION THREAD

**Discussion by x-ny-ng** [Security-Onion-Solutions/securityonion#12996] [2.4]
### Version

Other (please provide detail below)

### Installation Method

Network installation on Ubuntu

### Description

other (please provide detail below)

### Installation Type

Import

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4 (2*2)

### RAM

12GB

### Storage for /

unsure, I set 200GB for the VM configuration

### Storage for /nsm

unsure

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am currently facing the issue of an incomplete pcap file import. Using so-import-pcap, I am importing a 274kb file. However if I were to download the imported file from security onion, it is only 247kb. Looking at the respective pcap files, I noticed that for packets greater than the length of 1500, it gets truncated and it says 'packet size limited during capture. How do I import the whole pcap file without it getting truncated?

This screenshot shows the original pcap file.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/90682369/51b542aa-2280-4a6c-b763-243bcfa5e54d)

This screenshot shows the imported pcap file.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/90682369/090302bd-71be-4a00-b5f4-85d7b39771b7)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Are you able to share the original pcap?
